_examples/custom_http_auth/main.go

```diff
@@ -0,0 +1,59 @@
+package main
```

Comment:
I'm thinking we might want to rename this example from `custom_http_auth` to `http_basic_auth`